### DISCUSSION TITLE
Export to PDF

### DISCUSSION THREAD

**Discussion by ruben-vanwyk** [apache/superset#26148] [Q&A / Help]
Hi everyone. I know this question has been asked multiple times and recently there was a PR:
https://github.com/apache/superset/pull/25696
Just want to confirm if this change has been implemented? I recently downloaded and installed superset (from scratch) and still don't have the option to export to PDF?

**Comment by eschutho** [comment]
Hi @ruben-vanwyk, which version of Superset are you running? That feature will be released in 3.1.0 which is currently up for vote.

**Reply by ruben-vanwyk** [reply]
Hi @eschutho, thanks for the update. Yes, we've downloaded the latest 3.1.0 version (https://github.com/apache/superset/tree/3.1.0rc1) and set up the config file as instructed to 'activate' the feature but it still does not appear on the export menu?
